### PR TITLE
Replace ZEIT with Vercel in examples

### LINKED ISSUE #416: problem with one-click deploy in example code
The URL of one-click deploy has a problem. 
It shows error.

ERROR: The repository either does not exist or is not publicly accessible.

### PR DESCRIPTION
**Pull request by alexandregv**
Fixes one-click deploy links (fix #416)

### CODE REVIEW

File: examples/api-hooks/README.md [hunk 1]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/api-hooks/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/api-hooks/README.md [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/focus-revalidate)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/focus-revalidate)
```
Comment (pacocoursey) [inline examples/api-hooks/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/focus-revalidate)
```

File: examples/autocomplete-suggestions/README.md [hunk 3]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/autocomplete-suggestions/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/autocomplete-suggestions/README.md [hunk 4]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/autocomplete-suggestions)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/autocomplete-suggestions)
```
Comment (pacocoursey) [inline examples/autocomplete-suggestions/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/autocomplete-suggestions)
```

File: examples/axios-typescript/README.md [hunk 5]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/axios-typescript/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/axios-typescript/README.md [hunk 6]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/axios-typescript)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/axios-typescript)
```
Comment (pacocoursey) [inline examples/axios-typescript/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/axios-typescript)
```

File: examples/axios/README.md [hunk 7]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/axios/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/axios/README.md [hunk 8]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/axios)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/axios)
```
Comment (pacocoursey) [inline examples/axios/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/axios)
```

File: examples/basic-typescript/README.md [hunk 9]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/basic-typescript/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/basic-typescript/README.md [hunk 10]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/basic-typescript)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/basic-typescript)
```
Comment (pacocoursey) [inline examples/basic-typescript/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/basic-typescript)
```

File: examples/basic/README.md [hunk 11]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/basic/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/basic/README.md [hunk 12]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/basic)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/basic)
```
Comment (pacocoursey) [inline examples/basic/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/basic)
```

File: examples/focus-revalidate/README.md [hunk 13]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/focus-revalidate/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/focus-revalidate/README.md [hunk 14]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/focus-revalidate)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/focus-revalidate)
```
Comment (pacocoursey) [inline examples/focus-revalidate/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/focus-revalidate)
```

File: examples/global-fetcher/README.md [hunk 15]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/global-fetcher/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/global-fetcher/README.md [hunk 16]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/global-fetcher)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/global-fetcher)
```
Comment (pacocoursey) [inline examples/global-fetcher/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/global-fetcher)
```

File: examples/optimistic-ui-immer/README.md [hunk 17]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/optimistic-ui-immer/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/optimistic-ui-immer/README.md [hunk 18]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/optimistic-ui-immer)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/optimistic-ui-immer)
```
Comment (pacocoursey) [inline examples/optimistic-ui-immer/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/optimistic-ui-immer)
```

File: examples/optimistic-ui/README.md [hunk 19]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/optimistic-ui/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/optimistic-ui/README.md [hunk 20]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/optimistic-ui)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/optimistic-ui)
```
Comment (pacocoursey) [inline examples/optimistic-ui/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/optimistic-ui)
```

File: examples/pagination/README.md [hunk 21]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/pagination/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/pagination/README.md [hunk 22]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/pagination)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/pagination)
```
Comment (pacocoursey) [inline examples/pagination/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/pagination)
```

File: examples/prefetch-preload/README.md [hunk 23]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/prefetch-preload/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/prefetch-preload/README.md [hunk 24]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/prefetch-preload)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/prefetch-preload)
```
Comment (pacocoursey) [inline examples/prefetch-preload/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/prefetch-preload)
```

File: examples/storage-tab-sync/README.md [hunk 25]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/storage-tab-sync/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/storage-tab-sync/README.md [hunk 26]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/storage-tab-sync)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/storage-tab-sync)
```
Comment (pacocoursey) [inline examples/storage-tab-sync/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/storage-tab-sync)
```

File: examples/server-render/README.md [hunk 27]
```diff
@@ -2,16 +2,16 @@
 
 ## One-Click Deploy
 
-Deploy your own SWR project with ZEIT Now.
+Deploy your own SWR project with Vercel Now.
```
Comment (pacocoursey) [inline examples/server-render/README.md:5]:
```suggestion
Deploy your own SWR project with Vercel.
```

File: examples/server-render/README.md [hunk 28]
```diff
@@ -2,16 +2,16 @@
...
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/swr/tree/master/examples/server-render)
+[![Deploy with Vercel Now](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/server-render)
```
Comment (pacocoursey) [inline examples/server-render/README.md:7]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/project?template=https://github.com/vercel/swr/tree/master/examples/server-render)
```

**Comment by alexandregv** [comment]
Hadn't noticed this change, fixed

**Pull request merged**